<launch>
	<!-- declare arguments with default values -->
    <arg name="image_source_topic" default="/usb_cam/image_raw"/>
    <arg name="status_code_topic" default="/traffic/tls_code"/>
    <arg name="image_detected_topic" default="/traffic/image_detected"/>
	
    <node pkg="traffic_detect" type="traffic_detect" name="traffic_detect" args="$(find traffic_detect)/model/app.cfg" output="screen">   
    <param name="image_source_topic" value="$(arg image_source_topic)"/>
	<param name="status_code_topic" value="$(arg status_code_topic)"/>
    <param name="image_detected_topic" value="$(arg image_detected_topic)"/>
	</node>
</launch>
